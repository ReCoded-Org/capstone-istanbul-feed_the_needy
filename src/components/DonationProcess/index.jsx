import React from 'react';

const DonationProcess = () => <div>DonationProcess here!</div>;

export default DonationProcess;
