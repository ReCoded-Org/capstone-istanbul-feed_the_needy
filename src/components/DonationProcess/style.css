.donationProcessTitle {
  justify-content: center;
  text-align: center;
}

.donationProcessText {
  justify-content: center;
  text-align: center;
  width: 18em;
}
