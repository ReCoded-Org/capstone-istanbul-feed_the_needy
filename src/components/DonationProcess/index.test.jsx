import React from "react";
import renderer from "react-test-renderer";
import DonationProcess from "./index";

// Fix for TypeError: window.matchMedia is not a function
Object.defineProperty(window, "matchMedia", {
  writable: true,
  value: jest.fn().mockImplementation((query) => ({
    matches: false,
    media: query,
    onchange: null,
    addListener: jest.fn(), // deprecated
    removeListener: jest.fn(), // deprecated
    addEventListener: jest.fn(),
    removeEventListener: jest.fn(),
    dispatchEvent: jest.fn(),
  })),
});

jest.mock("react-i18next", () => ({
  useTranslation: () => ({ t: (key) => key }),
}));

it("Donation Process renders correctly", () => {
  const tree = renderer.create(<DonationProcess />).toJSON();
  expect(tree).toMatchSnapshot();
});
