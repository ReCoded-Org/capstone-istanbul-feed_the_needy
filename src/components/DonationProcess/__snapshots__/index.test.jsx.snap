// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Donation Process renders correctly 1`] = `
<div
  align="center"
  justify="center"
>
  <div
    className="ant-row ant-row-center donationProcessTitle"
    style={
      Object {
        "marginLeft": -8,
        "marginRight": -8,
      }
    }
  >
    <h1
      className="ant-typography"
      style={
        Object {
          "WebkitLineClamp": null,
        }
      }
    >
      donationProcessComp.card.title0
    </h1>
  </div>
  <div
    className="ant-row ant-row-space-around"
    style={
      Object {
        "marginLeft": -28,
        "marginRight": -28,
      }
    }
  >
    <div
      className="ant-col ant-col-xs-24 ant-col-sm-24 ant-col-md-12 ant-col-xl-8"
      style={
        Object {
          "paddingLeft": 28,
          "paddingRight": 28,
        }
      }
    >
      <div
        className="ant-card"
      >
        <div
          className="ant-card-body"
          style={Object {}}
        >
          <div
            className="donationProcessText"
          >
            <svg>
              stepDonation.svg
            </svg>
            <h2
              className="ant-typography"
              style={
                Object {
                  "WebkitLineClamp": null,
                }
              }
            >
              donationProcessComp.card.title1
            </h2>
            <h5
              className="ant-typography"
              style={
                Object {
                  "WebkitLineClamp": null,
                }
              }
            >
              donationProcessComp.card.title4
            </h5>
          </div>
        </div>
      </div>
    </div>
    <div
      className="ant-col ant-col-xs-24 ant-col-sm-24 ant-col-md-12 ant-col-xl-8"
      style={
        Object {
          "paddingLeft": 28,
          "paddingRight": 28,
        }
      }
    >
      <div
        className="ant-card"
      >
        <div
          className="ant-card-body"
          style={Object {}}
        >
          <div
            className="donationProcessText"
          >
            <svg>
              stepFood.svg
            </svg>
            <h2
              className="ant-typography"
              style={
                Object {
                  "WebkitLineClamp": null,
                }
              }
            >
              donationProcessComp.card.title2
            </h2>
            <h5
              className="ant-typography"
              style={
                Object {
                  "WebkitLineClamp": null,
                }
              }
            >
              donationProcessComp.card.title5
            </h5>
          </div>
        </div>
      </div>
    </div>
    <div
      className="ant-col ant-col-xs-24 ant-col-sm-24 ant-col-md-12 ant-col-xl-8"
      style={
        Object {
          "paddingLeft": 28,
          "paddingRight": 28,
        }
      }
    >
      <div
        className="ant-card"
      >
        <div
          className="ant-card-body"
          style={Object {}}
        >
          <div
            className="donationProcessText"
          >
            <svg>
              stepCoupon.svg
            </svg>
            <h2
              className="ant-typography"
              style={
                Object {
                  "WebkitLineClamp": null,
                }
              }
            >
              donationProcessComp.card.title3
            </h2>
            <h5
              className="ant-typography"
              style={
                Object {
                  "WebkitLineClamp": null,
                }
              }
            >
              donationProcessComp.card.title6
            </h5>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
